frontend/include/chpl/parsing/parser-error-classes-list.h

```diff
@@ -59,6 +59,7 @@ PARSER_SYNTAX_CLASS(ParseSyntax, std::string)
 
 /* begin post-parse-checks errors */
 POSTPARSE_ERROR_CLASS(CantApplyPrivate, std::string)
+POSTPARSE_ERROR_CLASS(WhenAfterOtherwise, const uast::Select*)
```

Comment:
Apparently `POSTPARSE_ERROR_CLASS` always includes an `AstNode`, so you don't have to pass it `Select` here. However, if you follow my recommendation and include both the `otherwise` and the offending `when` in this error, it would look like this:

```
POSTPARSE_ERROR_CLASS(WhenAfterOtherwise, const uast::When*, const uast::When*)
```